wordpress-mobile/WordPress-iOS
# Reader Comments: Implement comment rendering via WPRichContent

Refs #17476
Depends on #17634

As titled, this PR reintroduces the previous content rendering method through `WPRichContentView` into the comment thread.  Note that all pre-existing issues with the `WPRichContentView` (e.g. enlarged emojis, extra white spaces, partially clipped text) are not fixed here.

Here's a preview:
![Simulator Screen Shot - iPhone 12 - 2021-12-07 at 22 09 36](https://user-images.githubusercontent.com/1299411/145059161-1df55af6-f6f8-4846-9581-ff1db81fda9e.png)

---

I also fixed a minor visual issue regarding table view separators. Previously the separators were hidden – but now since we are showing a separator, the separators between empty cells show while the loading view is shown. This PR fixes that issue.

Before | After
--|--
![before](https://user-images.githubusercontent.com/1299411/145061521-83927ac7-5611-4dc8-a6b4-70e22a258c25.png) | ![after](https://user-images.githubusercontent.com/1299411/145061534-7d853db8-a485-4cdc-b7af-7b8d76044908.png)


## To test

- Ensure that the `newCommentThread` flag is enabled.
- Go to Reader > any comment thread.
- 🔍 Verify that the comment body is displayed with the old appearance, but contained in the new comment cell.
- 🔍 Tap on the overflow menu and verify that the menu items work.
- 🔍 Verify that the reply button works.
- 🔍 Verify that the like button works.
- Find a comment that contains a link in it.
- 🔍  Tap the link and verify that a modal window is presented and it loads the tapped URL.
- Find a comment that contains an image (or some).
- 🔍  Tap on the image and verify that a modal window is presented and it loads the image.

#### Verifying the table separator issue

- Go to Notifications, and tap any comment reply notification.
- Tap the notification header.
- 🔍 Verify that no table separators are shown while the loading view is visible. 

## Regression Notes
1. Potential unintended areas of impact
n/a. Feature is hidden behind a flag.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
n/a. Feature is hidden behind a flag.

3. What automated tests I added (or what prevented me from doing so)
n/a. Feature is hidden behind a flag.

PR submission checklist:
